src/Orleans/CodeGeneration/GeneratedAssembly.cs

```diff
@@ -32,5 +39,14 @@ public GeneratedAssembly(GeneratedAssembly other)
         /// Gets or sets a serialized representation of the assembly's debug symbol stream.
         /// </summary>
         public byte[] DebugSymbolRawBytes { get; set; }
+
+        /// <summary>
+        /// Gets or sets the assembly.
+        /// </summary>
+        public Assembly Assembly
+        {
+            get { return this.assembly; }
+            set { this.assembly = value; }
+        }
```

Comment:
The code generator is responsible for loading the generated assembly. This is where the result is stored.